akka-docs-dev/rst/scala/code/docs/stream/BidiFlowDocSpec.scala

```diff
@@ -50,7 +50,7 @@ object BidiFlowDocSpec {
     // construct and add the bottom flow, going inbound
     val inbound = b.add(Flow[ByteString].map(fromBytes))
     // fuse them together into a BidiShape
-    BidiShape(outbound, inbound)
+    BidiShape.fromFlows(outbound, inbound)
```

Comment:
Sounds good. As long as it is easy to get dot-completion/intellisense. I'll curate a proposed update on my PR: